src/luarocks/fetch/cvs.lua

```diff
@@ -20,9 +20,14 @@ function cvs.get_sources(rockspec, extract, dest_dir)
    assert(type(rockspec) == "table")
    assert(type(dest_dir) == "string" or not dest_dir)
 
+   local cvs_cmd = rockspec.variables.CVS
+   if not fs.execute_quiet(cvs_cmd, "--version") then
+      return nil, "'"..cvs_cmd.."' program not found. Is CVS installed? You may want to edit variables.CVS"
```

Comment:
Sure. I copied this message from the cmake builder. Considering that:
1) The user just wants to install a rock. She might have no idea what "hg" is.
2) The tool's executable may not indicate the tool's name (hg -> Mercurial)

What about:
'foo' program not found. Make sure Bar is installed and is available in your PATH"